adcm_aio_client/core/types.py

```diff
@@ -93,6 +93,9 @@ class MappingOperation(str, Enum):
     REMOVE = "remove"
 
 
+DEFAULT_JOB_TERMINAL_STATUSES: list[str] = ["broken", "aborted", "failed", "success"]
```

Comment:
Make it either `frozenset` (better) or `tuple` (ok): it shouldn't be mutable and search should be optimal.